### PR TITLE
chore: update libcosmic (+ other deps I think)

### LINKED ISSUE #22: [BUG] Text Not Properly Word-Wrapped in System Information Display
**Issue:**
On the system information screen, text in the right-side pane is not properly word-wrapped, causing it to display as a vertical line rather than wrapping to fit the container width. This layout issue makes the information difficult to read.

This issue is reproducible in each page with some param info

**Propose solution:**
- Implement word-wrap for entities on left so that when windows size is shrinked it makes space for information on right.
- Prevent shrinking window after certain point. Currently windows can be shrinked to a straight line

image

App Window when shrinked (It can further be shrinked)
image

### PR DESCRIPTION
**Pull request by sungsphinx**
Might fix #22 (sets minimum window size)

### CODE REVIEW

**Comment by ahoneybun** [comment]
Not sure if this is new or just how the page renders. 

screenshot-2024-11-26-16-18-52

**Comment by sungsphinx** [comment]
I think it was like that before or similar, not entirely sure though.

**Review by ahoneybun — APPROVED** [review]
Builds and works as expected.

File: src/app.rs [hunk 1]
```diff
@@ -557,13 +565,7 @@ pub enum ContextPage {
...
 }
 
-impl ContextPage {
-    fn title(&self) -> String {
-        match self {
-            Self::About => fl!("about"),
-        }
-    }
-}
+// impl ContextPage {}
```
Comment (edfloreshz) [inline src/app.rs]: This can be removed

File: src/app.rs [hunk 2]
```diff
@@ -59,14 +60,33 @@ impl Application for AppModel {
...
-            .set_application_icon(Self::APP_ID)
-            .set_developer_name("COSMIC Utilities")
-            .set_version(env!("CARGO_PKG_VERSION"))
-            .set_license_type("GPL-3.0")
-            .set_repository_url(REPOSITORY)
-            .set_support_url(format!("{REPOSITORY}/issues"))
-            .set_developers([("Dexter Reed".into(), "dreed4470@proton.me".into())]);
+            .name(fl!("app-title"))
+            .icon(Self::APP_ID)
+            .author("COSMIC Utilities")
```
Comment (edfloreshz) [inline src/app.rs]: You should be the author, we could add an organization method to link a project to an organization

**Review by edfloreshz — CHANGES_REQUESTED** [review]
Small changes needed, otherwise it looks good.

**Pull request merged**